app/src/pages/inside/settingsPage/settingsTabs/autoAnalysisTab/analysisForm/analysisForm.scss

```diff
@@ -0,0 +1,55 @@
+@import '../autoAnalysisTab';
```

Comment:
Is it necessary?

Reply:
I used classes such as .form-group-container, .form-group-column in both this components (analysisForm, accuracyFormBlock).
Rewrote specific elements classes. Fixed